### ISSUE TITLE
About Experimental environment

### ISSUE THREAD

**Issue by STORM-young** [jcwang123/xboundformer#5]
Hello author, I see that your code does not give the environment configuration part requirement .txt, I get an error when running your experiment showing RuntimeError: No CUDA GPUs are available, is it that I don't match your cuda version or something else?

**Comment by jcwang123** [comment]
Have you correctly installed the torch?

**Comment by STORM-young** [comment]
I installed it and its version is 1.13.1+cu117

**Comment by jcwang123** [comment]
That's correct. Can you run any other torch-based codes? 
The errors occur perhaps due to incorrect GPU settings. You could try to run the python file with "CUDA_VISIBLE_DEVICES=X"

**Comment by STORM-young** [comment]
Yes, the problem is caused by "CUDA_VISIBLE_DEVICES=X", I commented out(注释) this code and reminded me that the memory is insufficient, I will try to use 24G video memory to continue running this experiment in the follow-up, thank you. But just comment it out? Still need to be modified, I'm in a single GPU running experiment.

**Comment by jcwang123** [comment]
Instead of commenting out, You should change "X" to your available GPU device. maybe 0.

**Comment by STORM-young** [comment]
I have another question, are your results based on the 5-fold cross-validation results?   I run your code on the ISIC-2018 dataset.   In the training function, only iou and dice are output indexes, and assd and hd are not seen, and there is no obvious mark in the test function.   In which custom file are these two metrics calculated and output?
